src/pages/References.tsx

```diff
@@ -362,7 +362,23 @@ const ReferencesPage = () => {
                         }
                         url={"https://www.kali.org/tools/wpscan/"}
                     />
-
+                    <Reference
+                        name={"Sqlninja"}
+                        description={"Sqlninja is an SQL injection tool"}
+                        url={"https://www.kali.org/tools/sqlninja/"}
+                    />
+                    <Reference
+                        name={"Tiger"}
+                        description={"Toolset used to perform security audits of operating system components"}
+                        url={"https://www.kali.org/tools/tiger/"}
+                    />
+                    <Reference
+                        name={"Arping"}
+                        description={
+                            "Arping is a tool that sends ARP/ICMP requests to a target device and displays the reply messages"
```

Comment:
Full stop required. 
```suggestion
                            "Arping is a tool that sends ARP/ICMP requests to a target device and displays the reply messages."
```

Reply:
@michaeljpigott I've implemented the changes you recommended and made sure it passes all of the checks. It should be ready now.